Python/ast.c

```diff
@@ -112,6 +112,8 @@ validate_arguments(arguments_ty args)
 {
     if (!validate_args(args->args))
         return 0;
+    if (!validate_args(args->posonlyargs))
```

Comment:
Braces again, I'd also probably add them to the line above even though you didn't add it.